docs/team/vanessamae23.md

```diff
@@ -51,6 +51,7 @@ Given below are my contributions to the project.
 - Helped with other admin TP requirements.
 - Added test cases for my peers' code contribution.
 - Fixed bugs after the PE mock test
+- [Link to all my PR reviews](https://github.com/AY2324S1-CS2103T-T12-1/tp/pulls?q=is%3Apr+reviewed-by%3A%40me+is%3Aclosed)
```

Comment:
Wrong link - this shows all PR reviews by "me", which depends on the link opener. You should be specifying your own username.